### DISCUSSION TITLE
Inquiry 9

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#188] [Ideas]
Some ideas again.
1. Delay lobby join until we are the last person to join, perhaps bindable like anti-kick.
2. Damaging teammates with custom value and make it spammable ( e.g /damage or /dmg <teammate> <value> <duration>) speed of spam can be configured in file?

**Comment by winstxnhdw** [comment]
> Delay lobby join until we are the last person to join, perhaps bindable like anti-kick.

This is not possible because we can't know the state of the lobby if you aren't connected.

> Damaging teammates with custom value and make it spammable ( e.g /damage or /dmg ) speed of spam can be configured in file?

This is a great idea. I am thinking of something like `/poison` and you can inflict someone with a DoT. `/poison <player> <damage> <delay> <duration>`.

**Reply by winstxnhdw** [reply]
I've added this.

**Reply by yujispin990** [reply]
I saw it! thank you so much !!❤️

**Reply by yujispin990** [reply]
actually i was about to pair this with demigod, so i make my teammates demigods and constantly give 1 damage to them to annoy them XD.

**Reply by yujispin990** [reply]
with the added delay i can annoy them even more 😈

**Reply by yujispin990** [reply]
And i agree, poison is the best name for this feature!

**Comment by yujispin990** [comment]
> I've added this.
